test/http_content_length_test.dart

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import "dart:io";
+import "package:test/test.dart";
+
+void testNoBody(int totalConnections, bool explicitContentLength) {
```

Comment:
As originally written, these tests are a little broken because they all run concurrently. We need to make them run one at a time. We can do that by creating a completer, which is completed when the test is finished, whose future is returned by this function.

Reply:
Done.